agent/installer/cli.go

```diff
@@ -38,6 +38,17 @@ var (
 func Main() {
 	klogger = klogr.New()
 
+	listSupportedFlag = flag.Bool("list-supported", false, "List all supported OS, Kubernetes versions and BYOH Bundle names")
+	detectOSFlag = flag.Bool("detect", false, "Detects the current operating system")
+	installFlag = flag.Bool("install", false, "Install a BYOH Bundle")
+	uninstallFlag = flag.Bool("uninstall", false, "Unnstall a BYOH Bundle")
+	bundleRepoFlag = flag.String("bundle-repo", "projects.registry.vmware.com", "BYOH Bundle Repository")
+	cachePathFlag = flag.String("cache-path", ".", "Path to the local bundle cache")
+	k8sFlag = flag.String("k8s", "1.22.1", "Kubernetes version")
```

Comment:
@jeuxdemains @georgievaVMW 
Why is the default version `1.22.1` here? Why are we assuming any default version at all?

Reply:
I'm not sure why we have a default option. My assumption is that this is the minimal supported one and it defaults to it if it's missing when the CLI is invoked. 
Should we remove it?

Reply:
atm, I don't understand why we need a default version. Anyway, this is the CLI. We can discuss it on a different thread or tackle if this becomes an issue. Resolving this conversation. We are good with this PR.